### PR TITLE
Feat(Query): No New Privileges Not Set for docker compose

### PR DESCRIPTION
**Pull request by cxAndreFelicidade**
**Proposed Changes**
- Query: No New Privileges Not Set for docker compose

I submit this contribution under the Apache-2.0 license.

### CODE REVIEW

File: assets/queries/dockerCompose/no_new_privileges_not_set/query.rego [hunk 1]
```diff
@@ -0,0 +1,20 @@
+package Cx
+
+import data.generic.common as common_lib
+
+CxPolicy[result] {
+	resource := input.document[i]
+	service_parameters := resource.services[name]
+    sec_opt := service_parameters.security_opt
+    sec_opt[s] != "no-new-privileges:true"
+    sec_opt[s] != "no-new-privileges"
```
Comment (rafaela-soares) [inline assets/queries/dockerCompose/no_new_privileges_not_set/query.rego:10]: The policy is verifying each index of security_opt (and should not). I think we should look if there is no security_opt set to "no-new-privileges:true". 

```suggestion
    not no_new_privileges(sec_opt)
```

```
no_new_privileges(sec_opt) {
    sec_opt[_] == "no-new-privileges:true"
}
```

**Review by rafaela-soares — CHANGES_REQUESTED** [review]

**Review by rafaela-soares — APPROVED** [review]
LGTM 🚀

File: assets/queries/dockerCompose/no_new_privileges_not_set/test/positive_expected_result.json [hunk 2]
```diff
@@ -0,0 +1,14 @@
+[
+    {
+      "queryName": "No New Privileges Not Set",
+      "severity": "HIGH",
+      "line": 12,
+      "filename": "positive1.yaml"
```
Comment (joaorufi) [inline assets/queries/dockerCompose/no_new_privileges_not_set/test/positive_expected_result.json:6]:
```suggestion
      "fileName": "positive1.yaml"
```

File: assets/queries/dockerCompose/no_new_privileges_not_set/test/positive_expected_result.json [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+    },
+    {
+      "queryName": "No New Privileges Not Set",
+      "severity": "HIGH",
+      "line": 12,
+      "filename": "positive2.yaml"
```
Comment (joaorufi) [inline assets/queries/dockerCompose/no_new_privileges_not_set/test/positive_expected_result.json:12]:
```suggestion
      "fileName": "positive2.yaml"
```

**Review by joaorufi — CHANGES_REQUESTED** [review]

**Review by joaorufi — APPROVED** [review]

**Pull request merged**